app/lib/actions/katello/repository/capsule_sync.rb

```diff
@@ -12,7 +12,7 @@ def plan(repo)
               smart_proxies = ::SmartProxy.with_environment(repo.environment)
               unless smart_proxies.blank?
                 plan_action(::Actions::BulkAction, ::Actions::Katello::CapsuleContent::Sync, smart_proxies,
-                            :repository_id => repo.id)
+                            :repository_id => repo.id, :skip_metadata_check => false)
```

Comment:
do you remember what this was needed for?

Reply:
Removed this..I was trying something for repo sync and forgot to clear smart_proxy_sync_history when contents change upon sync..Updated.